manual/guides/pyne.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: manual
+title: PyNE
+subtitle: "An important tool in our toolkit"
+permalink: /manual/guides/pyne
+---
+
+# <a name="a brief overview"></a>A Brief Overview
+
+Taken from the [PyNE website](http://pyne.io/index.html), PyNE is intended to be a library used for nuclear engineering. It can be used to help its users "be more productive, have the *best* solvers, have a beautiful API, write really great code, and teach the next generation". The official website provides all the information that one would need, including information about [installation](http://pyne.io/install/index.html) and a [tutorial](http://pyne.io/tutorial/index.html) on using PyNE.
+
+# Development
+
+One of the perks of PyNE is that it is open source and free to use. Prof. Huff encourages contributions to PyNE, as its purpose is to provide a complete library meant to help all nuclear engineers. As such, anyone who feels that there should be something added or something changed is welcome to proceed in the proper manner to have their changed included in the final product, making their work easier. The website provides a [developer's guide](http://pyne.io/devsguide/index.html) intended to assist with testing, developing, and integration with PyNE. Prof. Huff is willing to help by offering some guidance on contributions, so any ideas on fixes or changes are encouraged.
```

Comment:
1. Once again, review the writing checklist.
2. This is a great place to link to the PyNE github.